reference/orders.v2.oas2.yml

```
@@ -4650,6 +4651,40 @@ components:
               description: The order product `id`. To add a product to an existing order, donʼt include `id` in the payload. `id` is required when updating an order product.
         - $ref: '#/components/schemas/orderCatalogProduct_Post'
       x-internal: false
+    orderRemoveProduct_Put:
+      title: Remove Product from Order
+      type: object
+      properties:
+        id:
+          type: integer
+          description: The `id` from the GET /stores/{store_hash}/v2/orders/{order_id}/products.
```

turn the endpoint reference into a link: 

```suggestion
          description: The `id` returned from a request to the [List order products](/docs/rest-management/orders/order-products#list-order-products) or the [Get an order product](/docs/rest-management/orders/order-products#get-an-order-product) endpoint.
```